### PR TITLE
Mercury 13.19.0 plat 7514

### PR DESCRIPTION
**Pull request by RoieBeck**

### CODE REVIEW

File: tests/standAloneClient/partnerAddedMediaSpaceHttpNotification.xml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            <template objectType="KalturaHttpNotificationTemplate">
+                <name>Partner Added MediaSpace</name>
+                <systemName>PARTNER_ADDED_MEDIASPACE</systemName>
+                <description>HTTP notification template to be sent to mediaSpace and partner will be created</description>
+                <eventType>5</eventType> <!-- EventNotificationEventType::OBJECT_CREATED -->
+                <eventObjectType>20</eventObjectType> <!-- EventNotificationEventObjectType::PARTNER -->
+                <automaticDispatchEnabled>1</automaticDispatchEnabled>
+                <eventConditions objectType="array">
+                    <item objectType="KalturaEventFieldCondition">
+                        <description>freeTrialAccountType is KMS</description>
```
Comment (MosheMaorKaltura) [inline tests/standAloneClient/partnerAddedMediaSpaceHttpNotification.xml]: rephrase

File: tests/standAloneClient/partnerAddedMediaSpaceHttpNotification.xml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                                explode(",", kConf::get('MediaSpace_Partner_Registration')['free_trial_account_types']) : array()))</code>
+                        </field>
+                    </item>
+                </eventConditions>
+                <contentParameters objectType="array">
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>alternative_url</key>
+                        <description>alternative url will override the url with the correct LB from local.ini</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>(kConf::hasParam('MediaSpace_Partner_Registration') ? 'http://' . kConf::get('MediaSpace_Partner_Registration')['internal_kms_hostname'] . '/latest/public/create_instance.php' : '')</code>
```
Comment (MosheMaorKaltura) [inline tests/standAloneClient/partnerAddedMediaSpaceHttpNotification.xml]: Move protocol type inside the url.

File: plugins/event_notification/providers/http/lib/model/HttpNotificationTemplate.php [hunk 3]
```diff
@@ -315,4 +316,16 @@ public function setSslKeyType($v)							{return $this->putInCustomData(self::CUS
 	public function setSslKey($v)								{return $this->putInCustomData(self::CUSTOM_DATA_SSL_KEY, $v);}
 	public function setSslKeyPassword($v)						{return $this->putInCustomData(self::CUSTOM_DATA_SSL_KEY_PASSWORD, $v);}
 	public function setCustomHeaders(array $v)					{return $this->putInCustomData(self::CUSTOM_DATA_CUSTOM_HEADERS, $v);}
+
+	private function processURL($getURL)
+	{
+		if(kDataCenterMgr::getCurrentDcId() == 1)
```
Comment (MosheMaorKaltura) [inline plugins/event_notification/providers/http/lib/model/HttpNotificationTemplate.php]: Take the name from kDataCenterMgr (need to add getName )

**Comment by RoieBeck** [comment]
@MosheMaorKaltura  reviewed

**Pull request merged**